src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import tasks.*;
```

Comment:
avoid importing the entire package (idk if this applies to your own class?), import specific modules from each package